### DISCUSSION TITLE
row out of order

### DISCUSSION THREAD

**Discussion by SudgeFunday** [AntlerVC/firetable#1296] [Support / Q&A] [answered]
hello

im working on a table and wen i try to populate data into it, i get some yellow dot next to it, that says row out of order, im wondering how this could be fixed?

im trying to create a project on flutterflow

<img width="1811" alt="Screenshot 2023-06-14 at 1 33 10 PM" src="https://github.com/rowyio/rowy/assets/89611733/82aaedc1-446b-4d87-aac4-cf35f006e929">
<img width="1786" alt="Screenshot 2023-06-14 at 1 33 02 PM" src="https://github.com/rowyio/rowy/assets/89611733/2cb720b2-1162-439c-8cbd-1e77b7bba063">

**Comment by BeeBombshell** [comment, marked as answer]
Hey @SudgeFunday! When you create a table with [random or a custom ID](https://docs.rowy.io/how-to/add-row#with-custom-id), by default the row will be displayed on the top while it is being edited. Later, when the table is refreshed, it will appear at a different position in the table, according to the generated Row ID.
